sdk/spring-experimental/spring-cloud-azure-native-configuration/pom.xml

```diff
@@ -147,6 +147,93 @@
     </plugins>
   </build>
 
+  <profiles>
+    <!-- Library cannot build for Java 11 and below -->
+    <!-- Spring Native requires Java11+ -->
+    <profile>
+      <id>java8</id>
+      <activation>
+        <jdk>[,11)</jdk>
```

Comment:
what is the meaning for this range? <11?

Reply:
Correct, this activate when the Java version is `< 11`